src/media/js/addon/actions/submit.js

```diff
@@ -131,7 +131,7 @@ export function create() {
       .then(res => {
         dispatch(submitOk(res.body));
       }, err => {
-        dispatch(submitError(err.response.body));
+        dispatch(validationFail(JSON.parse(err.response.text).detail));
```

Comment:
So we don't want to differentiate errors on validation and errors on creation. Should we port this to the initial submit logic as well?